### PR TITLE
Model: Add contacts attribute

### PR DESCRIPTION
**Pull request by Tex-Tang**

### CODE REVIEW

File: src/main/java/seedu/address/model/person/contact/Contact.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+ * Contact describes an attribute of a person with a link, value and logo.
+ */
+public abstract class Contact {
+
+    private final ContactType contactType;
+    private final String link;
+    private final String value;
+
+    /**
+     * Initialize a contact instance
```
Comment (wrewsama) [inline src/main/java/seedu/address/model/person/contact/Contact.java:15]: This comment should be in the present tense e.g. `Initialises...`

File: src/main/java/seedu/address/model/person/contact/Contact.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+     * @param link  Link to be used as hyperlink in UI.
+     */
+    public Contact(ContactType type, String value, String link) {
+        this.contactType = type;
+        this.value = value;
+        this.link = link;
+    }
+
+    /**
+     * Construct a contact instance
```
Comment (wrewsama) [inline src/main/java/seedu/address/model/person/contact/Contact.java:28]: Same as above

**Review by arnav-ag — APPROVED** [review]
LGTM

**Review by wrewsama — APPROVED** [review]
lgtm

**Pull request merged**